apps/src/StudioApp.js

```diff
@@ -1788,10 +1791,12 @@ StudioApp.prototype.updateHeadersAfterDropletToggle_ = function (usingBlocks) {
 /**
  * Handle updates after a droplet toggle between blocks/code has taken place
  */
-StudioApp.prototype.onDropletToggle_ = function () {
+StudioApp.prototype.onDropletToggle_ = function (dontAutofocus) {
   this.updateHeadersAfterDropletToggle_(this.editor.currentlyUsingBlocks);
   if (!this.editor.currentlyUsingBlocks) {
-    this.editor.aceEditor.focus();
+    if (!dontAutofocus) {
```

Comment:
Nit: I think I'd prefer to see this argument stated in the positive, and give it a default "true" value.
```javascript
StudioApp.prototype.onDropletToggle_ = function (autoFocus) {
  autoFocus = utils.valueOr(autoFocus, true);
  // ...
  if (autoFocus) {
    this.editor.aceEditor.focus();
  }
  // ...
};
```